.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -5342,6 +5342,7 @@ stages:
           --build-arg RUNTIME_IMAGE=$(runtimeImage) \
           --build-arg PUBLISH_FRAMEWORK=$(publishFramework) \
           --build-arg RELATIVE_PROFILER_PATH="$(relativeProfilerPath)" \
+          --build-arg RELATIVE_APIWRAPPER_PATH="$(relativeApiWrapperPath)" \
```

Comment:
What exactly is this?

Reply:
This is the path to the LD_PRELOAD wrapper library that's used by both the profiler and crash tracking. It is set in most dockerfiles where it's needed, but it was missing in this case. I need to figure out why the fact it was missing wasn't causing errors until recently!